AllASPFuncs.py

```diff
@@ -620,13 +620,23 @@ def constructImages(exposures, ra, dec, size = 7, background = False, roman_path
             im -= bg
             Lager.debug(f'Subtracted a background level of {bg}')
 
-        m.append(im.flatten())
-        err.append(err_cutout.flatten())
-        mask.append(np.zeros(size*size))
-        #w = (zero**2)*err_cutout.flatten()
+        #m.append(im.flatten())
```

Comment:
What's happening in this section?  Why are some lines commented out?  Is this something you're still deciding about?  Why does it say "for not"?

If you're not sure if this is the path you want, then you can write out each as a different case right here and have an option flag to trigger each behavior.

Otherwise, I suggest remove the commented-out lines.

If you're keeping the lines because you might want them someday, then make them accessible as something you can call with an option to active.  Commented-out means you have to edit the code to run that, which means you can't write a test that exercises the section against a specific version.